compiler_opt/rl/problem_configuration.py

```diff
@@ -95,6 +95,10 @@ def get_preprocessing_layer_creator(
   def get_runner(self, *args, **kwargs) -> compilation_runner.CompilationRunner:
     raise NotImplementedError
 
+  @abc.abstractmethod
+  def get_nonnormalized_features(self) -> Iterable[str]:
+    raise NotImplementedError
```

Comment:
can this have a default implementation returning []?

Reply:
Fixed. Also removed the `@abc.abstractmethod` decorator so that if it doesn't have to be overridden.